cfme/automate/explorer/method.py

```diff
@@ -301,8 +301,7 @@ def is_displayed(self):
         return (
             self.in_explorer and
             self.datastore.is_opened and
-            self.title.text == 'Editing Automate Method "{}"'.format(
-                self.context['object'].name) and
+            'Automate Method [{}'.format(self.context['object'].name) in self.title.text and
```

Comment:
This appears to weaken the is_displayed check, but it appears this view is used in one navigation destination. Although there is no explicit check for an 'edit(ing)' string/element on the page, the Add page title is markedly different. Inclusion of the method name in the title should be sufficient.